charts/substra-backend/values.yaml

```diff
@@ -315,3 +315,7 @@ securityContext:
   fsGroup: 1001
   runAsUser: 1001
   runAsGroup: 1001
+
+grpc:
+  keepalive:
+    timeMs: 120000
```

Comment:
As this is only a backend setting and not enforced globally (currently only the backend use GRPC but we never know) shouldn't this be located under the `backend:` key ?

Reply:
That makes sense. 

Note: We have plans to separate the event app from the backend. If we do so, then the key would have to be moved again.

Reply:
Yes, but if you move the event app out of the backend you would also need to move it from here to an `eventApp:` key !

Reply:
I agree with Alexandre points.
We may first need to put this key under `backend`

Reply:
https://github.com/SubstraFoundation/substra-backend/pull/319/commits/072bfbf83d2f1a66d11d88ce036e4014077f6f1e